apps/server/src/modules/video-conference/bbb/bbb.service.ts

```diff
@@ -86,12 +86,12 @@ export class BBBService {
 			.then((resp: AxiosResponse<string>) => {
 				const bbbResp = this.converterUtil.xml2object<BBBResponse<BBBBaseResponse>>(resp.data);
 				if (bbbResp.response.returncode !== BBBStatus.SUCCESS) {
-					throw new InternalServerErrorException(bbbResp.response.messageKey, bbbResp.response.message);
+					throw new InternalServerErrorException(`${bbbResp.response.messageKey}: ${bbbResp.response.message}`);
 				}
 				return bbbResp;
 			})
 			.catch((error) => {
-				throw new InternalServerErrorException(error);
+				throw error;
```

Comment:
The catches do nothing in this file, but InternalServerErrorException; looks correct for all this places.